src/Config_Command.php

```diff
@@ -163,6 +163,107 @@ public function create( $_, $assoc_args ) {
 	}
 
 	/**
+	 * Define a constant in the wp-config.php file.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [<name>]
+	 * : The constant name to be defined.
+	 *
+	 * [<value>]
+	 * : The constant value to be defined.
+	 *
+	 * [--add]
```

Comment:
What should it return the status code when `--add` will be given and the `name` already exists?
This PR returns `0`, I am thinking it should return `1`.